3p/iframe-messaging-client.js

```diff
@@ -100,6 +100,17 @@ export class IframeMessagingClient {
    */
   setupEventListener_() {
     listen(this.win_, 'message', event => {
+      try {
+        if (!this.hostWindow_.postMessage && this.hostWindow_.contentWindow) {
+          // this.hostWindow_ can now be set to an iframe, after it has been
+          // created but before it has finished loading. If we've gotten a
+          // message from that iframe, then it must exist, so its
+          // contentWindow is now non-null.
+          this.hostWindow_ = this.hostWindow_.contentWindow;
+        }
+      } catch (e) {
```

Comment:
This would occur if window is xdomained?  Do we want to warn in that case?  When would it happen?